torch/csrc/autograd/python_variable.cpp

```diff
@@ -289,54 +337,13 @@ static int THPVariable_clear(THPVariable* self) {
       }
     }
   }
+  // If we're removing the python reference to the C++ tensor, that C++ tensor
+  // better not have any uses.
+  TORCH_INTERNAL_ASSERT(self->cdata.unsafeIsBorrowed() || tensor.use_count() <= 1);
```

Comment:
How can the Tensor use count be `<1` here?

Reply:
I think you're right, see https://github.com/pytorch/pytorch/pull/75933#discussion_r852118527